src/apis/queryFunctions/apiClient.ts

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+
+const apiClient = axios.create({
+  // baseURL: `${SERVER_API_URL}`,
+  baseURL: process.env.NEXT_PUBLIC_SERVER_API_URL,
```

Comment:
```suggestion
  baseURL: `${process.env.NEXT_PUBLIC_SERVER_API_URL}/api`,
```
api가 중복되는 경로라서 추가해도 될 것 같아요!

Reply:
오 좋습니다!! bb